src/sqlfluff/rules/L016.py

```diff
@@ -429,7 +467,7 @@ def _eval(self, segment, raw_stack, **kwargs):
                     else:
                         break  # pragma: no cover
```

Comment:
Same comment here about a test case if you know how to add one through your understanding of this code.

Reply:
Resolving as per above.